lib/rdf/changeset.rb

```diff
@@ -111,7 +130,7 @@ def apply(mutable, **options)
     end
 
     ##
-    # @return [Boolean] `true` iff inserts and deletes are both empty
+    # @return [Boolean] +true+ iff inserts and deletes are both empty
```

Comment:
Why use `+true+`? It doesn't look like yard interprets this.

Reply:
I see the `.yardopts` in the root directory calls for the use of Markdown. I didn't know you could do this; indeed out of the box, YARD delimits code `+like this+`. That's fine; I'm more than content to change it.